farm/data_handler/processor.py

```diff
@@ -2057,6 +2071,12 @@ def _convert_answers(self, baskets):
                     label_idxs[0, :] = 0
                 else:
                     # For all other cases we use start and end token indices, that are relative to the passage
+                    if len(basket.raw["answers"]) > 1:
+                        logger.warning("More than one answer per question: be aware that multiple answers are not taken into consideration "
```

Comment:
I think having the statement in the docstrings is fine. Also this message is displayed too many times. I would like to see this message removed tbh.

Reply:
I understand that it is annoying to see the same message too many times, but how can I notify the user? maybe we can just leave the message warning that max_answers exceeded?

Reply:
Totally, the max_answers exceeded message is important and I would leave it. I just commented about the `if len(basket.raw["answers"]) > 1:` message. Could you remove this then?